content/terraform/v1.13.x/docs/language/block/resource.mdx

```diff
@@ -6,6 +6,12 @@ description: >-
 
 # `resource` block reference
 
+<Note>
+
+The beta release of Terraform 1.14 lets you trigger automations outside of Terraform with actions. Refer to the [Invoke an action](/terraform/language/v1.14.x/invoke-actions) beta documentation for more information.
```

Comment:
```suggestion
Terraform `v1.14.x (beta)` adds the `action_trigger` block to the `resource` block so that you can trigger actions  outside of Terraform CRUD operations. Refer to the [Invoke an action](/terraform/language/v1.14.x/invoke-actions) beta documentation for more information.
```
Should we link to the new version of this page instead?